pallets/loans/src/lib.rs

```diff
@@ -785,11 +785,12 @@ pub mod pallet {
             Self::update_reward_supply_index(asset_id)?;
             Self::distribute_supplier_reward(asset_id, &who)?;
 
-            T::Assets::transfer(asset_id, &who, &Self::account_id(), mint_amount, false)?;
-            Self::update_earned_stored(&who, asset_id)?;
-            let exchange_rate = Self::exchange_rate(asset_id);
+            let exchange_rate = Self::exchange_rate_stored(asset_id)?;
+            Self::update_earned_stored(&who, asset_id, exchange_rate)?;
```

Comment:
shall it be `earned_stored`

Reply:
Do you mean to rename update_earned_stored to earned_stored?